File: client_windows/SkyBlog/Api/api/article/model/Article.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Api.api.classify.model;
using Api.api.user.model;
using Api.model;

namespace Api.api.article.model
{
    public class Article : BaseModel
    {
        public int Id { get; set; }

        public string Title { get; set; }

        public string SubTitle { get; set; }

        public int AuthorId { get; set; }

        public User Author { get; set; }

        public int ClassifyId { get; set; }

        public Classify Classify { get; set; }

        public string Content { get; set; }

        public string CreateDate { get; set; }

        public string UpdateDate { get; set; }
    }
}

File: client_web_refactor/define/article.ts
import { IUser } from "@/define/user";
import { IClassify } from "@/define/classify";
import { IBaseEntity, IPageInfo } from "@/define/common";

export interface IComment extends IBaseEntity {
  content: string;
  authorId?: number;
  author?: IUser;
}

export interface IArticle extends IBaseEntity {
  title: string;
  subTitle: string;
  content?: string;

  authorId: number;
  author?: IUser;

  classifyId: number;
  classify?: IClassify;

  createDate?: string;
  updateDate?: string;
}

export interface IClassifyWithArticles {
  articles: IPageInfo<IArticle>;
  classify: IClassify;
}
